iron-lazy-page.html

```diff
@@ -0,0 +1,191 @@
+<!--
+Template element used in conjunction with `iron-lazy-pages`.
+
+@element iron-lazy-page
+-->
+<dom-module id="iron-lazy-page">
+</dom-module>
+<script>
+Polymer({
+  is: 'iron-lazy-page',
+
+  extends: 'template',
+
+  properties: {
+    /**
+     * The url corresponding to the content which should be lazy-loaded.
+     */
+    path: {
+      type: String,
+      value: ''
+    }
+  },
+
+  behaviors: [
+    Polymer.Templatizer
+  ],
+
+  /*
+    ##################
+    BEGIN DOM-IF LOGIC
+    ##################
+   */
+   _loadAndShow: function(cb, parent) {
+     var self = this;
+     var onImportFinished = function() {
+       cb(function() {
+         self._stamp(parent);
```

Comment:
It seems my suggestion caused a slightly different issue, now pages don't animate when they're initially stamped.
Revert my above suggestion I have a different solution that should cover both cases.